lib/src/Validation.cc

```diff
@@ -0,0 +1,44 @@
+#include <drogon/Validation.h>
+
+using namespace drogon;
+
+Validation::Validation() noexcept {
+    this->errorMessages = "";
+}
+
+Validation::~Validation() noexcept {  }
+
+bool Validation::validateAll() noexcept { 
+    return this->errorMessages == "";
+}
+
+string Validation::getMessages() noexcept { 
+    return this->errorMessages.substr(0, this->errorMessages.size() - 1);
```

Comment:
How about if the errorMessages is empty?

Reply:
nice, good, thky